crates/forge-runner/src/compiled_runnable.rs

```diff
@@ -24,7 +24,8 @@ pub struct TestCaseRunnable {
 
 #[derive(Debug, PartialEq, Clone, Deserialize, Default)]
 pub struct TestDetails {
-    pub entry_point_offset: usize,
+    #[serde(rename = "entry_point_offset")]
+    pub sierra_entry_point_instruction_idx: usize,
```

Comment:
```suggestion
    pub sierra_entry_point_statement_idx: usize,
```
I think we should consistently use the terminology:
- Sierra has statements
- CASM has instructions

Additionally, `statement_idx` postfix refers to compiler newtype `StatementIdx`